common/admin/telemetry/telemetry.go

```diff
@@ -0,0 +1,78 @@
+//go:build ceph_preview
+// +build ceph_preview
+
+package telemetry
+
+import (
+	"fmt"
+
+	ccom "github.com/ceph/go-ceph/common/commands"
+	"github.com/ceph/go-ceph/internal/commands"
+)
+
+// TelemetryStatus is the output of "ceph telemetry status"
+type TelemetryStatus struct {
+	Channel_basic        bool
+	Channel_crash        bool
+	Channel_device       bool
+	Channel_ident        bool
+	Channel_perf         bool
+	Contact              string
+	Description          string
+	Device_url           string
+	Enabled              bool
+	Interval             int
+	Last_opt_revision    int
+	Last_upload          string
+	Leaderboard          bool
+	Log_level            string
+	Log_to_cluster       bool
+	Log_to_cluster_level string
+	Log_to_file          bool
+	Organization         string
+	Proxy                string
+	Url                  string
+}
+
+func parseTelemetryStatus(res commands.Response) (*TelemetryStatus, error) {
+	m := &TelemetryStatus{}
+	if err := res.NoStatus().Unmarshal(m).End(); err != nil {
+		return nil, err
+	}
+	return m, nil
+}
+
+// GetTelemetryStatus returns the output of "ceph telemetry status"
```

Comment:
```suggestion
// TelemetryStatus returns ceph telemetry status information.
//
// Similar To:
//
//	ceph telemetry status
```

Reply:
Not possible, since the struct has that name already

Reply:
You're right! Thanks